pages/common/stack.md

```diff
@@ -25,3 +25,7 @@
 - Compile a project and execute a command after compilation:
 
 `stack build --exec "{{command}}"`
+
+- Run a program and pass an argument to it:
```

Comment:
Is this a haskell program that it runs?

Reply:
@sbrl Yes. It will run a program from `./stack-work` directory.

Reply:
Cool, thanks for the clarification!  Just wants to check I'm on the same page :-)